docker-app/qfieldcloud/core/models.py

```diff
@@ -691,6 +692,61 @@ def active_users(self, period_since: datetime, period_until: datetime):
             is_staff=False,
         ).filter(Q(id__in=users_with_delta) | Q(id__in=users_with_jobs))
 
+    def list_active_users_jobs_deltas_count(self) -> List[Dict[str, Union[str, int]]]:
+        """Return a List of Dictionaries mapping user ids into the following values:
+        - user_id
+        - username
+        - number of jobs scheduled by the user
+        - number of deltas scheduled by the user
+        """
+        delta = timedelta(weeks=4)
+        now = datetime.now()
+        four_weeks_ago = now - delta
+
+        # Get the organization's projects
+        organization_projects_pk = self.projects.all()
+
+        # Get the jobs & deltas by users in the target time-window
+        recent_jobs: QuerySet = (
+            Job.objects.filter(
+                project__in=organization_projects_pk,
+                created_at__gte=four_weeks_ago,
+                created_at__lte=now,
+            )
+            .values("created_by", "created_by__username")
+            .annotate(jobs_count=Count("created_by"), type=V("jobs"))
+            .order_by("-jobs_count")
+        )
+        recent_deltas: QuerySet = (
+            Delta.objects.filter(
+                project__in=organization_projects_pk,
+                created_at__gte=four_weeks_ago,
+                created_at__lte=now,
+            )
+            .values("created_by", "created_by__username")
+            .annotate(deltas_count=Count("created_by"), type=V("deltas"))
+            .order_by("-deltas_count")
+        )
+
+        # Join querysets' values
+        payload: Dict[str, Union[str, int]] = {}
+        for sched in chain(
```

Comment:
I would prefer `obj` . Eventhough it

I guess usually elsewhere it is the var name used:

```
for obj in some_queryset:
...
```